src/screens/FeedbackScreen.tsx

```diff
@@ -0,0 +1,217 @@
+import React , { useState } from 'react';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import { Text , StyleSheet , View , TextInput , TouchableOpacity , Modal , Pressable ,Keyboard, Dimensions , Image , ScrollView , TextStyle, ViewStyle , ImageStyle} from "react-native";
+import { screen_names } from '../constants/ScreenNames';
+import { ParamListBase } from '@react-navigation/native';
+import { fonts } from '../assets/fonts/fonts';
+import { colors } from '../constants/Colors';
+
+const FeedBackScreen = ({navigation}: NativeStackScreenProps<ParamListBase, screen_names.FEEDBACK, undefined>) : JSX.Element => {
+    const [modalVisible, setModalVisible] = useState(false);
+    const [text, onChangeText] = React.useState("");
+    const [selection, setSelection] = useState("");
+    const GreatEmoji = { emoji : "😊" , text : "Great" }
+    const OkayEmoji = { emoji : "😐" , text : "Okay" }
+    const BadEmoji =  { emoji : "😔" , text : "Bad"}
+    return (
+      <ScrollView>
```

Comment:
I have noticed the background colour `white` does not span the whole screen height?
**Suggestion** : give the scroll view background color white instead of the main_container

![photo_2022-10-10_13-15-25](https://user-images.githubusercontent.com/28436092/194844258-4c80463c-7331-40be-bb52-a628df2aa4c9.jpg)